src/tests/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests/Services/TestFunction.cs

```diff
@@ -35,10 +35,33 @@ public TestFunction(TestContext testContext, string hubName)
                     { "EnvironmentName", "LOCAL_ACCEPTANCE_TESTS" },
                     { "AzureWebJobsStorage", "UseDevelopmentStorage=true" },
                     { "NServiceBusConnectionString", "UseDevelopmentStorage=true" },
-                    { "ConfigNames", "SFA.DAS.EmployerIncentives" }
+                    { "ConfigNames", "SFA.DAS.EmployerIncentives" },
+                    { "ApplicationSettings:DbConnectionString", testContext.SqlDatabase.DatabaseInfo.ConnectionString }
                 };
 
-            _testContext = testContext;            
+            _testContext = testContext;
+
+            var paymentProfiles = new List<IncentivePaymentProfile>
```

Comment:
These paymentProfiles  should be set on the testContext  as test data set by the current test.